### PR TITLE
New Readme for 5.0

### PR DESCRIPTION
**Pull request by andersio**
This draft attempts to focus on what RAC itself offers with simple, short examples, so that it is easier to win hearts over (?). Would be glad to know if the format is acceptable.

(Builds are manually cancelled)

### CODE REVIEW

**Comment by andersio** [comment]
Hmm, it seems I should probably take away point 4 and 5, and expand on point 1-3. But I'd first like to hear you guys on the format.

File: README.md [hunk 1]
```diff
@@ -1,71 +1,120 @@
...
-
-If you have a question, please see if any discussions in our [GitHub
-issues](https://github.com/ReactiveCocoa/ReactiveCocoa/issues?q=is%3Aissue+label%3Aquestion+) or [Stack
-Overflow](http://stackoverflow.com/questions/tagged/reactive-cocoa) have already
-answered it. If not, please feel free to [file your
-own](https://github.com/ReactiveCocoa/ReactiveCocoa/issues/new)!
+## Introduction
 
-#### Compatibility
+__ReactiveCocoa__ wraps various aspects of Cocoa frameworks with the declarative primitives from [ReactiveSwift](). Let's go through a few core aspects of ReactiveCocoa:
```
Comment (mdiep) [inline README.md]: I'd just remove the `Let's go through` sentence. I don't think it adds much.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,120 @@
...
+	// wrapped object. It holds a weak reference to the wrapped object.
+	let property = DynamicProperty<String>(object: person,
+	                                       keyPath: #keyPath(person.name))
+	```
+
+But there are still more to be discovered, and more to be introduced. Read our in-code documentations and release notes to
+find out more.
+
+
+#### On ReactiveSwift
```
Comment (mdiep) [inline README.md]: I'd condense this section and put it up top.

File: README.md [hunk 3]
```diff
@@ -81,21 +130,28 @@ To add RAC to your application:
...
+
+1. The core Swift API - which provides FRP inspired primitives - had been spun off as [ReactiveSwift][].
+
+2. The legacy Objective-C API had been spun off as [ReactiveObjC][].
+
+3. The bridge between the core Swift API and the legacy Objective-C API is now hosted in ([ReactiveObjCBridge][]), which is mostly meant as a compatibility layer for older ReactiveCocoa projects.
+
+The Objective-C API will continue to be maintained and supported in the foreseeable future. But given the maturity of the framework and the language, non-trivial changes are not expected. Please visit [ReactiveObjC][] for more information.
+
+**We highly recommend all Swift projects using the Swift API.**
```
Comment (mdiep) [inline README.md]: This section could also be consolidated and put up top. I'm thinking that 1-2 paragraphs explaining ReacitveSwift and ReactiveObjC should be sufficient.

File: README.md [hunk 4]
```diff
@@ -1,74 +1,112 @@
...
+	Interactive UI components expose [`Signal`][]s for control events
+	and updates in the control value upon user interactions.
+	
+	A selected set of controls provide a convenience, expressive binding
+	API for [`Action`][]s.
+	
+	
+	```swift
+	// Update `allowsCookies` whenever the toggle is flipped.
+	perferences.allowsCookies <~ toggle.reactive.isOnValues
```
Comment (mdiep) [inline README.md]: _preferences_ is misspelled here. Sorry I missed this before!

Comment (andersio) [inline README.md]: 😛 Thanks for catching it.

**Pull request merged**